docs/source/conf.py

```diff
@@ -62,7 +63,7 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = []
+exclude_patterns = ['noaa_csp.rst']
```

Comment:
```suggestion
exclude_patterns = []
```
@weihuang-jedi Excluding this causes an error when building the docs because the doc appears in the table of contents.

Reply:
I was doing some test, but forgot to change back. Sorry for that.